src/tbp/monty/frameworks/models/evidence_matching.py

```diff
@@ -1763,9 +1763,9 @@ def _get_evidence_update_threshold(self, graph_id):
         ) and self.evidence_update_threshold.endswith("%"):
             percentage_str = self.evidence_update_threshold.strip("%")
             percentage = float(percentage_str)
-            assert (
-                percentage >= 0 and percentage <= 100
-            ), "Percentage must be between 0 and 100"
+            assert percentage >= 0 and percentage <= 100, (
```

Comment:
TIL that this can be rewritten as `0 <= percentage <= 100` and it gives the result you'd expect, with short-circuiting if the first comparison is False. I think I knew that already and forgot about it.

Changing it is outside the scope of this PR.

See https://docs.python.org/3/reference/expressions.html#comparisons